### ISSUE TITLE
Throttle down notification sounds

### ISSUE THREAD

**Issue by IvanSanchez** [vector-im/riot-android#920]
I run Riot on desktop and phone at the same time. 

If:
* Notifications on one-to-one chats are enabled, and
* I have riot-android on the background in the phone, and
* I have riot-electron on foreground in the desktop computer, and
* Somebody is chatting with me
Then:
* I will get a notification with sound for each time the other person messages me, regardless of the fact that I've already read that in the desktop computer

Thus, feature request: Keep the first notification as they're now, but throttle down any later ones. The technical implementation may be setting the [`FLAG_ONLY_ALERT_ONCE` bit when instantiating the notification](https://developer.android.com/reference/android/support/v4/app/NotificationCompat.html#FLAG_ONLY_ALERT_ONCE), or maybe something else like lowering the priority.

**Comment by ylecollen** [comment]
it is easy to satisfy everybody. I understand your use case
->  we try to have the same behaviour on each platform (IOS / Android / web)
-> Do you expect having notifications when there is a notified message in another room ?

**Comment by IvanSanchez** [comment]
> Do you expect having notifications when there is a notified message in another room ?

Yes, I'd like that. I would expect _one_ notification sound _per room_, ideally.
